docs/source/markdown/podman-create.1.md

```diff
@@ -18,6 +18,10 @@ any point.
 
 The initial status of the container created with **podman create** is 'created'.
 
+Default settings for flags are defined in `containers.conf`. Most settings for
```

Comment:
```suggestion
Default settings for options are defined in `containers.conf`. Most settings for
```

Reply:
Down below you have "Default setting are defined..." If you wanted to drop "for flags" here entirely, that would work too.